CMakeLists.txt

```diff
@@ -6,9 +6,19 @@ cmake_minimum_required(VERSION 3.15)
 set(EVT_CORE_DIR      ${CMAKE_SOURCE_DIR}/libs/EVT-core)
 
 include(CMakeDependentOption)
+
+option(NUCLEO_COMPILATION
+        "Set this to compile the code to run on the Nucleo development board"
+        OFF
+        )
+if(NUCLEO_COMPILATION)
+    add_definitions(-DNUCLEO_COMPILATION)
```

Comment:
you should be able to simply have `cmake -DNUCLEO_COMPILATION ../` when you build and not have to add an additional cmake option

Reply:
Thanks.  I didn't realize that was an option.

Reply:
Following the output you sent
cmake ../ -DNUCLEO_COMPILATION=